### ISSUE TITLE
Create a user journey test from sending a mentorship request and the user deletes the request.

### ISSUE THREAD

**Issue by rpattath** [anitab-org/mentorship-backend#911]
## Description

I need to test a specific user path,
so that I can make sure a specific path can be executed usefully in a deterministic way.

This test should be located in a specific folder for separation concerns, it could be called tests/user_journey.
Next, I'll define a very broad scenario, but simple enough for you to start.

https://github.com/anitab-org/mentorship-backend/blob/develop/tests/user_journey/test_happy_path_1.py can be used as a reference but make sure you include all possible test scenarios and not just the happy path.

## Scenario: User A (mentor) and User B (mentee)

    1. User B (mentee) sends a request to User A (mentor)
    2. User B (mentee) deletes request

## Definition of Done
- [ ] Create tests for for positive and negative scenarios.

## Estimation
3 hours

**Comment by faznan3nazer** [comment]
@rpattath  Hey... I would like to work on this...can I claim this?

**Comment by faznan3nazer** [comment]
@rpattath I am still working on this issue...will open PR in 2 days

**Comment by abhinav72610** [comment]
@faznan3nazer  if you are not working on this can i work on this

**Comment by vj-codes** [comment]
@faznan3nazer hey how's the PR going? Any updates?

**Comment by faznan3nazer** [comment]
> @faznan3nazer hey how's the PR going? Any updates?

@vj-codes  I am working on this issue and will create PR in this week.

**Comment by sankha555** [comment]
@vj-codes Shall I take up this issue if you think I can be assigned? I have created a similar patch before for a rejection journey too and will be able to make this is in a jiffy. Awaiting your thoughts, thanks!

**Comment by faznan3nazer** [comment]
@vj-codes This is my first try at coding PR... I am still figuring out to run this... so I just want to let you know that I am still working on this issue and I will update my status every two days...

**Comment by vj-codes** [comment]
@faznan3nazer hey no problem take your time:)
If you need any help do message on zulip 😃 
@sankha555 this issue is not available , do go through other available issues 🚀

**Comment by decon-harsh** [comment]
@vj-codes If this is available I would love to give this a try.

**Comment by decon-harsh** [comment]
pinging again @vj-codes

**Comment by vj-codes** [comment]
@decon-harsh sure you can, the issue should be made available due to inactivity but I don't have access to assign you at this moment .
@isabelcosta @aditmehta9 can you please assign @decon-harsh ?

**Comment by isabelcosta** [comment]
@faznan3nazer do you still want to work on this issue?

**Comment by vj-codes** [comment]
Unassigning @faznan3nazer due to inactivity and there was no response to the above message nor any progress was updated here.
@decon-harsh are you interested to work on this?

**Comment by epicadk** [comment]
@decon-harsh would you like to work on this issue?

**Comment by decon-harsh** [comment]
@vj-codes @epicadk Yes i would love to work on this , However i am bit preoccupied with my personal works . How about i send the PR in two days ? Will that work.

**Comment by epicadk** [comment]
@decon-harsh Sure two days works :). 
Assigning you. Happy testing!
